syz-manager/manager.go

```diff
@@ -1066,7 +1061,143 @@ func (mgr *Manager) fuzzerConnect() ([]rpctype.RPCInput, BugFrames, bool) {
 	for frame := range mgr.dataRaceFrames {
 		dataRaceFrames = append(dataRaceFrames, frame)
 	}
-	return corpus, BugFrames{memoryLeaks: memoryLeakFrames, dataRaces: dataRaceFrames}, mgr.coverFilterFilename != ""
+	var err error
+	mgr.once.Do(func() {
```

Comment:
We don't need once now. This function is protected by mgr.mu and we have modulesInitialized flag, so we can do just:
```
if !modulesInitialized { ... }
```